core/src/scene/sceneLoader.cpp

```diff
@@ -661,7 +667,7 @@ void SceneLoader::loadTexture(const std::shared_ptr<Platform>& platform, const s
     float density = 1.f;
     if (Node d = textureConfig["density"]) {
         double val;
-        if (getDouble(d, val)) { density = val; }
+        if (YamlUtil::getDouble(d, val, false)) { density = val; }
```

Comment:
Though implicit cast works, but probably makes sense to be explicit and verbose here and use `YamlUtil::getFloat`